cloudify/_compat.py

```diff
@@ -29,6 +29,7 @@
 def reraise(exception_type, value, traceback):
     raise exception_type, value, traceback
 """)
+    text_type = unicode
```

Comment:
in `cloudify._compat` we do use the stricter approach with `text_type = unicode`; only in dsl-parser, we have the more lenient `text_type = (unicode, str)` that the commit message talks about (and hopefully we'll drop it there as well soon)